### PR TITLE
Problem: wrong /var/motr links created if restore executed from secon…

### PR DESCRIPTION
**Pull request by mssawant**
## Problem Statement
<pre>
  <code>
Problem: wrong /var/motr links created if restore executed from secondary node
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
prepare_var_motr_dirs() is also executed to restore /var/motr directories
post node replacement. But the function presently assumes it to be executed
from primary node only which may not be the case during restore as restore
can be executed from secondary node as well. This creates wrong /var/motr
links.
Also motr systemd unit file is not updated on restore.
  </code>
</pre>
## Solution
<pre>
  <code>
- Run /var/motr creation commands over ssh for both the nodes to handle the
  execution from any of the nodes.
- Update motr systemd unit on restore as well.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
 Server failback must work post node restore operation.
  </code>
</pre>

### CODE REVIEW

**Review by andriytk — APPROVED** [review]

**Review by indrajitzagade — APPROVED** [review]
I think this is merge to main branch from cortx-1.0 branch.

**Comment by andriytk** [comment]
It is, yes.

File: conf/script/build-ha-io [hunk 1]
```diff
@@ -260,19 +260,19 @@ net_config_check() {
 # rvolume is mounted at /var/motr2.
 prepare_var_motr_dirs() {
     log "${FUNCNAME[0]}: make sure there are no stale mounts hanging..."
-    mountpoint -q /var/motr1 && sudo umount /var/motr1 || true
+    ssh $lnode 'mountpoint -q /var/motr1 && sudo umount /var/motr1 || true'
```
Comment (ajaysrivas) [inline conf/script/build-ha-io:263]: why not use run_on_both function for all of such commands?

Comment (andriytk) [reply conf/script/build-ha-io:263]: Because the commands are different?

**Comment by andriytk** [comment]
This is the merge of #167 to the main branch. (Just for the record.)

**Pull request merged**